services/server/config/config.go

```diff
@@ -255,25 +255,30 @@ func loadConfigFile(path string) (*Config, error) {
 // - Non abs path is relative to parent config file directory
 // - Abs paths returned as is
 func resolveImports(parent string, imports []string) ([]string, error) {
-	var out []string
+	paths := make(map[string]struct{})
 
 	for _, path := range imports {
+		path = filepath.Clean(path)
+		if !filepath.IsAbs(path) {
+			path = filepath.Join(filepath.Dir(parent), path)
+		}
```

Comment:
This looks like a breaking change, the original logic has been changed here.
`Non-absolute paths containing *` will match from the parent config directory, not the current working directory.